<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>konfiguracjaBazy</class>
 <widget class="QDialog" name="konfiguracjaBazy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_konfiguracja_bazy">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Nazwa_serweraK">
       <property name="text">
        <string>Nazwa serwera</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Nazwa_serweraK"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Nazwa_bazyK">
       <property name="text">
        <string>Nazwa bazy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Nazwa_bazyK"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_LoginK">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_LoginK"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_HasloK">
       <property name="text">
        <string>Has≈Ço</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_HasloK"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_Testuj">
       <property name="text">
        <string>Zapisz</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_polaczonoK">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p/&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
